solr/modules/extraction/src/test/org/apache/solr/handler/extraction/TestXLSXResponseWriter.java

```diff
@@ -38,8 +38,10 @@
 import org.apache.solr.search.SolrReturnFields;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore("Crave tests fails on headless jvm")
```

Comment:
I'll remove this `@Ignore` before commit.

Reply:
Maybe you don't recall but we use the string "nocommit" to convey that.  `gradlew precommit` will fail so we won't forget.